commands/root.go

```diff
@@ -28,12 +29,45 @@ func NewRootCmd(name string, isPlugin bool, dockerCli command.Cli) *cobra.Comman
 		}
 	}
 
-	logrus.AddHook(logutil.NewFilter(
+	logrus.SetFormatter(&logutil.Formatter{})
```

Comment:
Why are the lines 32-51 needed?

Reply:
```go
logrus.SetFormatter(&logutil.Formatter{})
```

Logs don't match the output format from docker/cli, it feels weird:

```shell
$ docker build ...
WARNING: Error loading config file: ...
WARN[0000] No output specified for docker-container driver. Build result will only remain in ...
```

So use a simple formatter for logrus for parity:

```shell
$ docker build ...
WARNING: Error loading config file: ...
WARNING: No output specified for docker-container driver. Build result will only remain in ...
```

Ofc we could remove all `logrus.Warn|Debug|...` in our code and replace them with a simple printf but it's easier to use a formatter.

For lines 34-55 we are now using the docker/cli writers for logrus.

Reply:
> For lines 34-55 we are now using the docker/cli writers for logrus.

What's the benefit of that?

Reply:
For the warnings, there is also a case of the build warnings that have their own layout. Not strictly against the formatting change.

Reply:
> What's the benefit of that?

Just to be aligned with the cli. If you feel it's not worth it I can remove this commit.

> For the warnings, there is also a case of the build warnings that have their own layout. Not strictly against the formatting change.

I think it's ok as you're not using the logrus formatter right?: https://github.com/docker/buildx/blob/11b771c789daa8d6c9bc3cf7ba7c86240f382bef/commands/build.go#L268-L306

Reply:
> I can remove this commit.

I think so if it doesn't change anything.

> I think it's ok as you're not using the logrus formatter right?:

Yes. Just I can see it being confusing for the user.

Reply:
Done